release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -929,6 +929,18 @@ See link:https://access.redhat.com/articles/6955985[Navigating Kubernetes API de
 With this release, replacement control plane nodes are assigned to the correct instance group.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1970464[*BZ#1970464*], link:https://issues.redhat.com/browse/OCPCLOUD-1562[*OCPCLOUD-1562*])
 
+* Previously, a compute machine set for Google Cloud Platform could attempt to reconcile invalid machines, causing them to be stuck with no phase assigned. With this release, machines with an invalid configuration are put into the `Failed` state. (link:https://issues.redhat.com/browse/OCPBUGS-4574[*OCPBUGS-4574*])
+
+* Previously, a control plane machine set replica was considered ready when its backing machine entered the `Running` state, even though the linked node must also be ready for the replica to be considered ready. With this release, a node and its machine must be in the `Ready` state for the control plane machine set replica to be considered ready. (link:https://issues.redhat.com/browse/OCPBUGS-8424[*OCPBUGS-8424*])
+
+* Previously, the `mapi_instance_create_failed` alert metric did not fire when there was an error for the Accelerated Networking feature on Microsoft Azure clusters. This release adds the missing alert so that clusters with Accelerated Networking enabled generate alerts when required. (link:https://issues.redhat.com/browse/OCPBUGS-5235[*OCPBUGS-5235*])
+
+* Previously, when a machine entered the `Running` state, no further changes to the state of its node were checked for. The previous resolution of link:https://issues.redhat.com/browse/OCPBUGS-8424[OCPBUGS-8424] introduced the requirement for a node and its machine to be in the `Ready` state for the control plane machine set replica to be considered ready. As a result, if the control plane machine set missed the stage when the node and machine were ready, its replica could not become ready. This caused the Control Plane Machine Set Operator to become unavailable, blocking upgrades. With this release, when a machine is running but the node is not ready, the node is checked at regular intervals until it becomes ready. This prevents the Control Plane Machine Set Operator from becoming unavailable and blocking upgrades. (link:https://issues.redhat.com/browse/OCPBUGS-10771[*OCPBUGS-10771*])
```

Comment:
Per [IBM Style](https://www.ibm.com/docs/en/ibm-style?topic=grammar-pronouns#ambiguous-pronoun-references), try to always include a noun after "This" to avoid an ambiguous pronoun reference.

Reply:
Nice catch, thank you. @lahinson